_source/_posts/mhe-nmds/2008-01-01-data-entry.md

```diff
@@ -7,6 +7,7 @@ directory:
     name: MHE NMDS Data Entry Tool
 last_modified: 1
 category: mhe-nmds
+permalink: /mhe-nmds-data-entry-tool/2007-2008/
```

Comment:
Since the `/mhe-nmds-data-entry-tool/` prefix doesn't change is it possible to avoid having to repeat this in each entry. For example, if the posts where in an `/mhe-nmds-data-entry-tool/`directory rather than `mhe-nmds` would they get the prefix automatically?

Reply:
No it won't work. When I rename the directory to `/mhe-nmds-data-entry-tool/` and set `permalink: /2007-2008/`, the directory will be `http://localhost:4000/2017-2018/`.

Reply:
@thvu11, look at the the [posts2collection](https://github.com/strategicdata/validator.com.au/tree/feature/posts2collection) branch for a way to do this better. Posts are the default kind of collection and are work best for blog style posts. By using the more generic form you get more control.

Note that the menu automation will need to be updated to handle this.